jekyll/_docs/npm-continuous-deployment.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: classic-docs
+title: "Continuous deployment with npm"
+short-title: "npm publish from CircleCI"
+description: "How to configure CircleCI to publish packages to npm automatically"
```

Comment:
npm here refers to the npm registry, but I agree people may be confused. I’ll use “npmjs.org” to be more precise.